src/js/events/MesosStateActions.js

```diff
@@ -37,8 +51,31 @@ var MesosStateActions = {
       };
     },
     {delayAfterCount: Config.delayAfterErrorCount}
-  )
+  ),
 
+  fetchNodeState: RequestUtil.debounceOnError(
+    Config.getRefreshRate(),
+    function (resolve, reject) {
+      return function (task, node, success, error) {
+        return RequestUtil.json({
+          url: getNodeStateURL(task, node),
+          success(response) {
+            resolve();
+            success(response);
```

Comment:
This is deviating from Flux. Curious why we can't rely on the AppDispatcher to emit the results?